### ISSUE TITLE
Add dedicated changelog

### ISSUE THREAD

**Issue by Karneades** [alexandreborges/malwoverview#38]
Hi @alexandreborges,

Nice project :+1: I was looking at your project today and I have a small documentation proposal.  I suggest having a dedicated changelog for the project. Would you be open to have one like it is documented in [Keep a Changelog](https://keepachangelog.com)? It's easier to follow the changes and have a quick overview of the current changes, than scrolling through the readme, and it allows linking directly to the corresponding version's section.

What do you think?

Cheers
Andreas

**Comment by alexandreborges** [comment]
Dear Andreas, 

Good morning (or evening). How are you?

Your suggestion is very interesting and I will consider to add a dedicated change log. Please, give me some time to read about it.

Thank you for the message and good suggestion.

Have an excellent day.

Alexandre.

**Comment by Karneades** [comment]
Hi Alexandre,

Cool, thanks to consider that. I hacked together a new changelog up until version 5.0.0 to show you how it could look like and help you a bit. Please look at it in my `changelog` branch on my fork: https://github.com/Karneades/malwoverview/blob/changelog/CHANGELOG.md.

I will make a PR and add the rest of the history, and you could link that file in your README later. Or you can just copy it to your project without PR. What works best for you. 

Cheers
Andreas

**Comment by alexandreborges** [comment]
Andreas, 

Good  evening. How are you?

First, I am sorry for the huge delay in replying your message, but lots of thing have happened to my life since when you sent the message. 

I checked your repository and I really liked so much. Another professional has helped the project by doing a refactoring it, so we will release the version 6.0.0 soon. Thus, I like to release the new version first and, afterwards, accept your PR to complement this issue about the version tracking. Would be it possible?

Once again, I apologize for the delay in answering the message.

I hope you have an excellent day.

Best regards,

Alexandre.

**Comment by Karneades** [comment]
Hey Alexandre,

> Good evening. How are you?
> First, I am sorry for the huge delay in replying your message, but lots of thing have happened to my life since when you sent the message.

Nice to hear back from you! I'm fine and no worries at all because of answering not until now. Thanks for your response and despite your message, I hope you are well!

> Thus, I like to release the new version first and, afterwards, accept your PR to complement this issue about the version tracking. Would be it possible?

Sounds cool with the new release and of course, that's fine. If I can help in anyways, then just ping me.

**Comment by alexandreborges** [comment]
Hi Andreas, 

Good morning. How are you?

As I had promised you, I would come back to talk about your idea on the Change Log. Unfortunately, the new version took a long time due to my complete migration to another area (exploit development and vulnerability research), and also the the restrict time from another contributor to develop the project. Certainly I know that our conversation did happen 14 months, but sometime things don't move on as we expected. 

Finally, the version 6.0.0 has been released, and if you have interest to submit a proposal about the Change Log, we have a dev branch that you might clone, test, submit to the same dev branch and, if everything is working well, so we can migrate to the main branch.  

I hope you are doing well, healthy and have an excellent day.

Cheers, 

Alexandre.
